docs/docs/reference/inference.md

```diff
@@ -0,0 +1,57 @@
+# Inference
+
+## Helpers
+
+::: sbi.inference.base.infer
+
+::: sbi.utils.user_input_checks.prepare_for_sbi
```

Comment:
this was deprecated in https://github.com/sbi-dev/sbi/pull/1013
can you please add `process_prior` and `process_simulator`? 🙏